Automattic/wp-calypso
# Store Orders: Add fees to single order views

This PR adds rows to the order table to display the fees on an order. On the single order view:

<img width="735" alt="screen shot 2017-10-10 at 10 17 45 pm" src="https://user-images.githubusercontent.com/541093/31419288-018847d4-ae09-11e7-93eb-af85599a0e91.png">

This is also added to the refunds modal:

<img width="737" alt="screen shot 2017-10-10 at 10 18 03 pm" src="https://user-images.githubusercontent.com/541093/31419289-01b1b862-ae09-11e7-8877-5728d625c505.png">

Additionally this adds (and uses) some helper functions to `woocommerce/lib/order-values`, with tests:

- `getOrderFeeTax`
- `getOrderFeeTotalTax`
- `getOrderFeeTotal`

Note: This PR uses the new refund process in #18728.

**To test**

- View a single order with fees (can add by editing an on-hold or pending order in wp-admin)
- Open the refund modal by clicking "Submit Refund" (might have to Mark as Paid first)
- Make sure the fee displays correctly in both places, and the refund total is as expected
- Test out the new values helper functions: `npm run test-client client/extensions/woocommerce/lib/order-values/test/index.js`

@kellychoffman @jameskoster I went off the design in the order edit mockup, but I'm not sure about in the refund dialog.
